com.github.childishgiant.mixer.yml

```diff
@@ -18,6 +18,7 @@ command: com.github.childishgiant.mixer
 # not using hardware like webcams, making sound, or reading external files, we
 # only need permission to draw our app on screen using either X11 or Wayland.
 finish-args:
+  - '--device=all'
```

Comment:
Why is device=all added?

Reply:
Mixer is not changing sound settings without this permisson.

I will try with 

> --socket=pulseaudio - Access to PulseAudio, includes sound input (mic), sound output/playback, MIDI and ALSA sound devices in /dev/snd

Reply:
Mixer is not changing sound settings without this permission on my system.

Reply:
It's been literal years since I worked on this project really but I remember that I ended up needing that for it to work :/